en/style/typography.md

```diff
@@ -30,42 +34,64 @@ These styles also determine the type of text element and typography class that w
   - `igx-typography__caption` for the caption typography style.
   - `igx-typography__overline` for the overline typography style.
 
-These styles can be modified further via the Sketch plugin to change the look of those elements via theming on the application level.
+These styles can be modified further via the **Indigo.Design** plugin to change the look of those elements via theming on the application level.
 
 Use these when creating intricate layouts for articles, blog posts etc. The Text Styles in the **Indigo.Design System** are meant to help you define a consistent theme and style for all the texts in your designs.
 
 > [!Warning]
-> In Sketch, there is also a collection of `Internal` text styles used within the Components for validation and special cases. These styles are not supposed to be inserted on your artboards directly but exist to assure typographic consistency for the Components within which they are used.
+> In Figma and Sketch, there is also a collection of `Internal` text styles used within the Components for validation and special cases. These styles are not supposed to be inserted on your artboards directly but exist to assure typographic consistency for the Components within which they are used.
 
 ## Component Specific Typography
 
 Components, such as [Avatar](../components/avatar.md), [Button](../components/button.md), [Grid](../components/grid.md), [Slider](../components/slider.md) and [Tooltip](../components/tooltip.md) use component-specific Typography to accommodate the specifics of the styling used by the respective component e.g. the Avatar with initials and Button use an all caps style. Below you can find the text styles defined for the Avatar in Sketch to allow more versatile usage of font colors.
```

Comment:
This info is a bit inaccurate again, because in Figma most of these aren't defined as styles. I think only the Grid styles - detail 1 and 2 are defined as styles.

Reply:
Ok, I will fix it right after specifying what I should add